redbot/cogs/audio/apis.py

```diff
@@ -15,11 +16,15 @@
     from databases import Database
 
     HAS_SQL = True
-except ModuleNotFoundError:
+    _ERROR = None
+except (ModuleNotFoundError, ImportError) as err:
+    _ERROR = "".join(traceback.format_exception_only(type(err), err)).strip()
+    _ERROR_ARGS = f"Arguments:\n{err.args!r}"
```

Comment:
I don't think this is needed here, it looks like a debug message but it's not referenced anywhere in the code (and the format it would use if it were sending this to user doesn't seem very user-friendly due to using repr, but I don't know what you had in mind :)).

Reply:
Yes this was debug my bad